README.md

```diff
@@ -138,14 +138,26 @@ Paths provided are relative to the current directory from which you execute the
 
 There are plenty of examples you can use to start off:
 
-- `example`: contains an example app with version 2 of the specification. It will give you an idea how to annotate your comments in order to include them in the output specification.
+- `examples/js`: contains an example app with version 2 of the specification. It will give you an idea how to annotate your comments in order to include them in the output specification.
+- `examples/ts`: contains an example TypeScript app with version 2 of the specification. 
 - `test/cli`: CLI tests you can read to get ideas about the available functionalities of the CLI. (apart from the obvious help menu)
 - `test/example`: various assets for tests you can also re-use for starting definitions, routes, etc.
 
 ### CLI
 
 On top of the Node API, there is also a [command line interface](./docs/CLI.md).
 
+
+### Using with TypeScript at Runtime
+
+Yes, you can use swagger-jsdoc with your TypeScript project. There are some adjustments in the configuration necessary, though.
+1. You can install types via `@types/swagger-jsdoc`.
+2. Typescript is usually transpiled to JavaScript. So in order to generate the Swagger document the `apis: [...]` option has to point to the transpiled JavaScript files, and not to the TypeScript source files. In default configuration, the transpiled files are located in the `dist` directory (and the files end with `.js`).
+3. Annotating the model classes with the swagger comments is a great idea. However, these comments have to survive the transpilation process. A short workaround is to add a dummy function that you annotate instead of the model classes. 
```

Comment:
As suggested in https://github.com/Surnet/swagger-jsdoc/pull/195/files#r416776025 suggestions here should rather be:
- check TS configuration for comments stripping, i.e. `--removeComments` from https://www.typescriptlang.org/docs/handbook/compiler-options.html
- run `swagger-jsdoc` on raw source (.ts) rather than production files in order to use the comments and later let-go for production
- run `swagger-jsdoc` on transpiled source and double-check all necessary files and references are included in glob patterns 

I have to be honest that I haven't had the opportunity to use swagger-jsdoc in production projects i've been working on, however, if i did, i'd surely try to:
1) try my best to create the open api specification before the code goes for CI/CD or production. Can potentially save compute time, easier to debug locally rather than on continous systems with less permissoins to read logs, etc. and see missing references and similar already mentioned
2) if the transpilation is to happen on production, glob patterns matching needs to be treated with extra care at 120%. I understand people who want to have they actual code handled direclty on production in terms of gathering open api spec information, etc., but making mistakes with slow patterns or having commetns in production code is less than ideal rathre than preparing the spec before code goes into prod
3) making spec in run-time in production is obviously the least ideal when all the disadvantages of the previous are combined.

--removeComments
--